opm/grid/cpgrid/processEclipseFormat.cpp

```diff
@@ -231,6 +231,14 @@ namespace cpgrid
             // Make the grid.
             processEclipseFormat(g, nnc_cells, z_tolerance, false, turn_normals);
         }
+
```

Comment:
Should we maybe deactivate this test in production by `#ifndef NDEDUG`

Reply:
Better safe than sorry - I would prefer to leave that test in - even in release mode; can add a commented of `#ifndef NDEBUG` - which can be activated at some point in the future.